inst/scripts/make-grieneisen-ts-metadata.R

```diff
@@ -0,0 +1,70 @@
+
+library(dplyr)
+library(R.utils)
+library(stringr)
+
+BiocVersion <- "3.14"
+path <- paste0("MicrobiomeDataSets/",BiocVersion,"/")
+
+df_Base <- data.frame(
+  BiocVersion = BiocVersion,
+  SourceVersion = NA,
+  Coordinate_1_based = TRUE,
+  Domain = "Bacteria",
+  TaxonomyId = "2",
+  SourceVersion=Sys.time(),
+  Genome = NA,
+  SourceUrl = "https://science.sciencemag.org/content/373/6551/181",
+  DataProvider = NA
+)
+
+df <- rbind(
+  cbind(df_Base,
+       data.frame(
+                  Title = "Counts",
+                  Description = paste0("Count matrix for the bacteria microbiome dataset"),
+                  SourceType = "Rds",
+                  RDataClass = "DFrame",
+                  DispatchClass = "Rds",
+                  RDataPath = paste0(path,"counts.rds"),
+                  Tags = "Microbiome" )),
+  cbind(df_Base,
+        data.frame(
+                  Title = "Row data",
+                  Description = paste0("Taxonomy table for the bacteria microbiome dataset"),
+                  SourceType = "Rds",
+                  RDataClass = "DFrame",
+                  DispatchClass = "Rds",
+                  RDataPath = paste0(path,"rowdata.rds"),
+                  Tags = "Microbiome")),

```

Comment:
I am not sure what should be included in Tags. Do you find any documentation on this from ExperimentHub instructions? The other metadata scripts have NA here, not sure if this was intended (@FelixErnst ?)

Reply:
They have NA there, but this is added later on, Compare to 

https://github.com/microbiome/microbiomeDataSets/blob/04452ec9fcb53ff30d4b5bf7693441145c8c0b24/inst/scripts/make-lahti-ml-metadata.R#L58